src/defaultConfig.ts

```diff
@@ -81,6 +81,7 @@ export const defaultSecurityConfig = (serverlUrl: string): ModuleOptions => ({
     exclude: [/node_modules/, /\.git/]
   },
   ssg: {
-    hashScripts: true
+    hashScripts: true,
```

Comment:
suggestion: Could you please write some documentation for both hashScripts and hashStyles?

I think either `utils/3.ssg.md` or `getting-started/2.configuration.md`